### PR TITLE
cifsd: fix some get_info and set_info implemenations

### PR DESCRIPTION
**Pull request by tymok**
cifsd: fix some get_info and set_info implemenations

### CODE REVIEW

File: smb2pdu.c [hunk 1]
```diff
@@ -4413,6 +4415,8 @@ int smb2_get_info_filesystem(struct smb_work *smb_work)
 		return rc;
 	}
 
+	logical_bytes_per_sector =
+		get_logical_sector_size(path.dentry->d_inode);
```
Comment (namjaejeon) [inline smb2pdu.c]: this code should move to switch case. so this line should be called in FS_SIZE_INFORMATION and FS_SECTOR_SIZE..

File: smb2pdu.c [hunk 2]
```diff
@@ -4541,17 +4545,23 @@ int smb2_get_info_filesystem(struct smb_work *smb_work)
...
 
 			ss_info = (struct smb3_fs_ss_info *)(rsp->Buffer);
+			physical_bytes_per_sector =
+				get_physical_sector_size(path.dentry->d_inode);
+			optimal_io_size =
+				get_optimal_io_size(path.dentry->d_inode);
 
 			ss_info->LogicalBytesPerSector =
-				cpu_to_le32(bytes_per_sector);
+				cpu_to_le32(logical_bytes_per_sector);
```
Comment (namjaejeon) [inline smb2pdu.c]: logical_bytes_per_sector and physical_bytes_per_sector can properly renamed logical_sector_size and phyical_sector_size.

File: vfs.c [hunk 3]
```diff
@@ -1309,3 +1309,43 @@ unsigned short get_logical_sector_size(struct inode *inode)
 
 	return ret_val;
 }
+
```
Comment (namjaejeon) [inline vfs.c]: how about make get_sector_size_info function to get 3 value (logical, physical, optimal size) ?

**Comment by tymok** [comment]
Thanks for the review. I applied your review opinion of #17 and #19 now. Please check them.

**Pull request merged**